app/src/main/java/com/lastaoutdoor/lasta/repository/db/UserDBRepository.kt

```diff
@@ -45,6 +45,18 @@ interface UserDBRepository {
    */
   suspend fun addFavorite(userId: String, activityId: String)
 
+  /**
+   * Gets Users corresponding having in their username the given string keyword.
+   *
+   * @param query the string to search for in the username

```

Comment:
you forgot to doc the second and third parameter ^^

Reply:
yes thanks for noting I just updated it